gamebench_api_client/api/requests_retriever/builder/url/url_builder.py

```diff
@@ -83,7 +83,10 @@ def set_url(self):
     def _set_session_id(self, session_id):
         """ Helper method to set the session_id. """
 
-        self.session_id = session_id
+        if session_id != '':
```

Comment:
Hey Tristan,
This particular code block is confusing me a bit. The docstring is partial, and I'm not sure I understand the choice to use this code:
`if session_id != '':
            self.session_id = '/' + session_id
        else:
            self.session_id = session_id`
Instead of something like:
`if session_id:
    self.session_id = session_id
else:
    self.session_id = '/' + session_id
'

Reply:
Thanks for the code suggestion, I do agree your suggestion is cleaner!  I've updated the if statement, and updated the docstring.

If the client passed a session id with a slash already in it then yes it would add it.  We could add some logic to verify the session id is alpha-numeric.  If we wanted to do that, should we have it closer to the layer where the ID is passed in before it even gets past the ModelCreator?